src/stores/subscribable.ts

```diff
@@ -87,11 +87,12 @@ export function gqlSubscribable<T>(
    */
   function getTokenFromUserCookie(): string {
     if (browser && document?.cookie) {
-      const userCookie = document.cookie.split('user=')[1];
-
+      const cookies = document.cookie.split(/\s*;\s*/);
```

Comment:
@duranb or @jeffpamer Do you know of any nice front-end libs that can do this for us?

Reply:
There's [cookie-parser](https://expressjs.com/en/resources/middleware/cookie-parser.html), but that feels like overkill for what we need it for.